OpenRA.Mods.Cnc/Activities/AttackIgnoringVisibility.cs

```diff
@@ -0,0 +1,178 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Activities;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Cnc.Activities
+{
+	/* Attack against targets that might not be visible (i.e. Hunter-Seeker) */
+	public class AttackIgnoringVisibility : Activity, IActivityNotifyStanceChanged
```

Comment:
```suggestion
	public class AttackIgnoringVisibility : Activity, IActivityNotifyStanceChanged
```

That is Imho obvious from the name. That being said, can we not re-use one of the existing attack activities? This looks like a bunch duplicated code from a quick glance.